Casks/r/ripme.rb

```diff
@@ -1,16 +1,22 @@
 cask "ripme" do
-  version "2.1.12-7-d0b97acd"
-  sha256 "87671af2600aca3374295680ac2b7ffbc3aed5eadff85669e995e584fc8b29da"
+  version "2.1.13-7-fac3f8ea"
+  sha256 "196e0048aaf742ac44840675b46907502a144a0b3d27402e7666c29e94aea185"
 
-  url "https://github.com/RipMeApp/ripme/releases/download/#{version}/ripme-#{version}-0.jar"
+  url "https://github.com/RipMeApp/ripme/releases/download/#{version}/ripme-#{version}.jar"
   name "RipMe"
   desc "Album ripper for various websites"
   homepage "https://github.com/RipMeApp/ripme"
 
+  livecheck do
+    url :url
+    regex(/v?(\d+(?:\.\d+)+-\d+-\h+)/i)
```

Comment:
```suggestion
    regex(/v?(\d+(?:\.\d+)+(?:-\d+-\h+)?)/i)
```

The upstream repository has used standard version tags as well, so we should make the suffix optional.